github/compliance/model.py

```diff
@@ -101,14 +107,25 @@ class ScanResultGroup:
     def results_with_successful_scans(self):
         return tuple(r for r in self.results if r.scan_succeeded)
 
+    @property
+    def target(self) -> Target:
```

Comment:
consistently call it `scanned_element`, as in the underlying results. is this property actually required?

Reply:
It's a simple shortcut to the very first result (`results[0].scanned_element`) but its used multiple times.
It follows the same semantics as `@property component` or `@property artifact`.

Reply:
yeah, this is pretty hacky (has been before already). which operations are done when it is used? maybe it would be better to offer shortcuts to those functions, instead.

Reply:
A `ScanResultGroup.target` is used as an representative `ScanResult.scanned_element` to get common things across `ScanResult`s in a `ScanResultGroup`.
Its used e.g. to determine responsibles or reporting repository for a `ScanResultGroup`.

We could consider adding those functions directly to the model class or add shortcuts, but in the end the underlaying hacky part with all its implications remains.